cmd/limactl/main.go

```diff
@@ -55,8 +55,22 @@ func newApp() *cobra.Command {
 		SilenceUsage:  true,
 		SilenceErrors: true,
 	}
+	rootCmd.PersistentFlags().String("log-level", "", "Set the logging level [trace, debug, info, warn, error, fatal, panic]")
+	rootCmd.PersistentFlags().Bool("silent", false, "silent mode")
 	rootCmd.PersistentFlags().Bool("debug", false, "debug mode")
 	rootCmd.PersistentPreRunE = func(cmd *cobra.Command, args []string) error {
+		l, _ := cmd.Flags().GetString("log-level")
+		if l != "" {
+			lvl, err := logrus.ParseLevel(l)
+			if err != nil {
+				return err
+			}
+			logrus.SetLevel(lvl)
+		}
+		silent, _ := cmd.Flags().GetBool("silent")
+		if silent {
+			logrus.SetLevel(logrus.WarnLevel)
```

Comment:
In many unix utilities `silent` implies no output at all. Curl for instance wouldn't give you anything other than the output from the command.